vega/static/vega-lite.js

```diff
@@ -2,6 +2,10 @@ var spec = {spec};
 var selector = "{selector}";
 var type = "{type}";
 
+console.log(spec);
+console.log(selector);
+console.log(type);
+
```

Comment:
Do you mean for these to stay in here? Some sort of permanent console message might actually be useful to keep.